figures/views.py

```diff
@@ -322,6 +323,31 @@ def retrieve(self, request, *args, **kwargs):
         return Response(GeneralCourseDataSerializer(course_overview).data)
 
 
+class CourseTopStatsViewSet(CommonAuthMixin, viewsets.ReadOnlyModelViewSet):
+    """
+    Viewset to get top courses by enrollments/completions.
+    """
+    model = CourseDailyMetrics
+
+    # The "kilo paginator"  is a tempoarary hack to return all course to not
+    # have to change the front end until Figures "Level 2"
+    pagination_class = FiguresKiloPagination
+    serializer_class = CourseTopStatsSerializer
+
+    def get_queryset(self):
+        site = django.contrib.sites.shortcuts.get_current_site(self.request)
+        course_ids = figures.sites.get_course_keys_for_site(site)
+        queryset = self.model.objects.filter(course_id__in=course_ids)
+        order_by = self.request.query_params.get('order_by', '')
+        if order_by:
+            order_by_name = order_by.split(',')[0]
```

Comment:
what if `order_by ` is empty? this will raise key not found exception.

Reply:
if `order_by` is empty, it wouldn't go in the if block as `''` translates to false. This is the same way it has been done in the edly panel app as well.